doc/EFFECT_ON_CONDITION.md

```diff
@@ -2267,6 +2267,66 @@ Pick all items with `RECHARGE` _or_ `ELECTRONIC` flags, and run `EOC_PRINT_ITEM_
 }
 ```
 
+#### `u_map_run_eocs`, `npc_map_run_eocs`
+Picks all tiles around you, npc or target_var, stores it's coordinates in `store_coordinates_in`, and then checks EoC conditions for each tile picked
+Used if you need to check if specific furniture or terrain is around
+
+| Syntax | Optionality | Value  | Info |
+| --- | --- | --- | --- | 
+| "u_map_run_eocs", "npc_map_run_eocs" | **mandatory** | string, [variable object](#variable-object) or array | EoC or EoCs that would be run |
+| "store_coordinates_in" | optional | [variable object](#variable-object) | variable, where testet coordinate is stored | 
```

Comment:
```suggestion
| "store_coordinates_in" | optional | [variable object](#variable-object) | variable, where tested coordinate is stored | 
```
Is only one set of coordinates stored in the variable?

Reply:
it doesn't store set per se, more lile `pick first tile, write it's coordinates in variable, check condition. pick second tile, write it's coordinates in variable, check condition. pick third tile...`